server/Gemfile.lock

```diff
@@ -150,6 +151,7 @@ DEPENDENCIES
   faye-websocket (~> 0.10.0)
   httpclient (~> 2.3)
   json-jwt (= 1.5.2)
```

Comment:
(I have a hunch that this json-jwt thing is a remnant from some auth experiments)